x-pack/plugin/sql/sql-proto/src/main/java/org/elasticsearch/xpack/sql/proto/SqlVersions.java

```diff
@@ -0,0 +1,212 @@
+    public static final SqlVersion V_8_3_2 = fromId(8_03_02_99);
+    public static final SqlVersion V_8_3_3 = fromId(8_03_03_99);
+    public static final SqlVersion V_8_4_0 = fromId(8_04_00_99);
+    public static final SqlVersion V_8_4_1 = fromId(8_04_01_99);
+    public static final SqlVersion V_8_4_2 = fromId(8_04_02_99);
+    public static final SqlVersion V_8_4_3 = fromId(8_04_03_99);
+    public static final SqlVersion V_8_5_0 = fromId(8_05_00_99);
+    public static final SqlVersion V_8_5_1 = fromId(8_05_01_99);
+    public static final SqlVersion V_8_5_2 = fromId(8_05_02_99);
+    public static final SqlVersion V_8_5_3 = fromId(8_05_03_99);
+    public static final SqlVersion V_8_6_0 = fromId(8_06_00_99);
+    public static final SqlVersion V_8_6_1 = fromId(8_06_01_99);
+    public static final SqlVersion V_8_6_2 = fromId(8_06_02_99);
+    public static final SqlVersion V_8_7_0 = fromId(8_07_00_99);
+    public static final SqlVersion V_8_7_1 = fromId(8_07_01_99);
+    public static final SqlVersion V_8_8_0 = fromId(8_08_00_99);
+    public static final SqlVersion V_8_8_1 = fromId(8_08_01_99);
+    public static final SqlVersion V_8_8_2 = fromId(8_08_02_99);
+    public static final SqlVersion V_8_9_0 = fromId(8_09_00_99);
+    public static final SqlVersion V_8_9_1 = fromId(8_09_01_99);
+    public static final SqlVersion V_8_9_2 = fromId(8_09_02_99);
+    public static final SqlVersion V_8_10_0 = fromId(8_10_00_99);
+    public static final SqlVersion V_8_10_1 = fromId(8_10_01_99);
+    public static final SqlVersion V_8_10_2 = fromId(8_10_02_99);
+    public static final SqlVersion V_8_10_3 = fromId(8_10_03_99);
+    public static final SqlVersion V_8_10_4 = fromId(8_10_04_99);
+    public static final SqlVersion V_8_11_0 = fromId(8_11_00_99);
+    public static final SqlVersion V_8_11_1 = fromId(8_11_01_99);
+    public static final SqlVersion V_8_11_2 = fromId(8_11_02_99);
+    public static final SqlVersion V_8_11_3 = fromId(8_11_03_99);
+    public static final SqlVersion V_8_11_4 = fromId(8_11_04_99);
+    public static final SqlVersion V_8_12_0 = fromId(8_12_00_99);
+    public static final SqlVersion V_8_12_1 = fromId(8_12_01_99);
+    public static final SqlVersion V_8_12_2 = fromId(8_12_02_99);
+    public static final SqlVersion V_8_13_0 = fromId(8_13_00_99);
+    public static final SqlVersion V_8_13_1 = fromId(8_13_01_99);
+    public static final SqlVersion V_8_13_2 = fromId(8_13_02_99);
+    public static final SqlVersion V_8_13_3 = fromId(8_13_03_99);
+    public static final SqlVersion V_8_13_4 = fromId(8_13_04_99);
+    public static final SqlVersion V_8_14_0 = fromId(8_14_00_99);
+    public static final SqlVersion V_8_14_1 = fromId(8_14_01_99);
+    public static final SqlVersion V_8_14_2 = fromId(8_14_02_99);
+    public static final SqlVersion V_8_14_3 = fromId(8_14_03_99);
+    public static final SqlVersion V_8_15_0 = fromId(8_15_00_99);
+    public static final SqlVersion V_8_15_1 = fromId(8_15_01_99);
```

Comment:
How will this be updated for further 8.15 patch releases?

Reply:
It wouldn't be updated anymore. All 8.15.x release will behave as current 8.15.1.

Reply:
Does it mean that if we have to patch 8.15 (pretty unlikely), we won't need a new version here?